packages/wdio-cucumber-framework/src/utils.js

```diff
@@ -6,7 +6,10 @@ export function createStepArgument ({ argument }) {
     if (argument.type === 'DataTable') {
         return [{
             rows: argument.rows.map(row => (
-                { cells: row.cells.map(cell => cell.value) }
+                {
+                    cells: row.cells.map(cell => cell.value),
+                    locations: row.cells.map(cell => cell.location)
```

Comment:
@christian-bromann why do we need locations here?

Reply:
I think it can be value-able for reporters